client/src/pages/HomeView/styles.css

```diff
@@ -0,0 +1,11 @@
+body {
+  background-color: rgb(255, 250, 250);
+}
+
+.title {
+  padding-top: 100px;
```

Comment:
We want to use `rem` or `%` rather than `px` where possible to accommodate for different display sizes.

Reply:
Navbar is fixed height – so accounting for that by padding. (Honestly, will probably define a layout component in the future that accounts for that and is applied to all the pages)